### PR TITLE
Adds addressRegion to the address claim table for international addresses

### PR DESCRIPTION
**Pull request by Caitlin-cooling**
**Do not merge this PR until we've got approval from the policy board that we can go ahead**

## Why

To ensure the docs about VCs reflect changes for international addresses

## What

Adding `addressRegion` to address claim table

## Technical writer support

Yes

## How to review

Changes can be viewed locally at http://localhost:4567/integrate-with-integration-environment/prove-users-identity/#understand-your-user-s-address-claim

## Confirm

- [x] I have checked if any docs change here also requires updates to other repositories (ADRs / RFCs, README.md, Team Manual, elsewhere in these docs)
- [x] Where there is any overlap I have updated or opened a PR for corresponding changes

### CODE REVIEW

File: source/integrate-with-integration-environment/prove-users-identity.html.md.erb [hunk 1]
```diff
@@ -296,6 +296,7 @@ Each JSON object in the list may contain any of the following properties:
...
 | `postalCode`                     | Maps to `POST_CODE` in the [Postcode Address File][external.postcode-file] and [Ordnance Survey Places API][external.os-places-api].                                                                                                                                                                                          |
 | `addressCountry`                 | Two-letter [ISO 3166-1 alpha-2 country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2).                                                                                                                                                                                                                               |
+| `addressRegion`                  | The region in which the locality is, and which is in the country, provided as text. For example, California, or another appropriate first-level Administrative division.                                                                                                                                                      |
```
Comment (CoralSmith) [inline source/integrate-with-integration-environment/prove-users-identity.html.md.erb:299]: Could we include something to say that it aligns to https://schema.org/addressRegion? eg.
```suggestion
| `addressRegion`                  | The region in which the locality is, and which is in the country, provided as text. For example, California, or another appropriate first-level Administrative division.  This maps to [schema:addressRegion](https://schema.org/addressRegion)                                                                                                                                                   |
```

Comment (Caitlin-cooling) [reply source/integrate-with-integration-environment/prove-users-identity.html.md.erb]: Done

**Review by philf999 — DISMISSED** [review]
Looks good to me and the inclusion of the maps to keeps it consistent with the other attributes.

**Review by SteveEvans-GDS — CHANGES_REQUESTED** [review]

File: source/integrate-with-integration-environment/prove-users-identity.html.md.erb [hunk 2]
```diff
@@ -296,7 +296,7 @@ Each JSON object in the list may contain any of the following properties:
...
 | `postalCode`                     | Maps to `POST_CODE` in the [Postcode Address File][external.postcode-file] and [Ordnance Survey Places API][external.os-places-api].                                                                                                                                                                                          |
 | `addressCountry`                 | Two-letter [ISO 3166-1 alpha-2 country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2).                                                                                                                                                                                                                               |
-
+| `addressRegion`                  | Maps to [schema:addressRegion](https://schema.org/addressRegion). Only returned for international addresses and will contain the region, provided as text. For example, California or another appropriate first-level Administrative division.                                                                                |
```
Comment (ImogenCraigmile) [inline source/integrate-with-integration-environment/prove-users-identity.html.md.erb:299]: Looks good to me, just querying if the capital A is necessary on Administrative division - I've had a google and I've seen it lower case and upper, is there a reason why you opted for upper case?

**Review by ImogenCraigmile — COMMENTED** [review]
Just the one query about Administrative division but otherwise looks really good!

File: source/integrate-with-integration-environment/prove-users-identity.html.md.erb (hunk 2 above)
Comment (SteveEvans-GDS) [reply source/integrate-with-integration-environment/prove-users-identity.html.md.erb:299]: That's how it's referenced on the schema.org page: https://schema.org/addressRegion, which is where the info for this column comes from.

**Review by ImogenCraigmile — APPROVED** [review]
LGTM

**Review by SteveEvans-GDS — APPROVED** [review]

**Pull request merged**